docs/content/commands/rclone_cryptdecode.md

```diff
@@ -40,4 +40,5 @@ See the [global flags page](/flags/) for global options not listed here.
 ## SEE ALSO
 
 * [rclone](/commands/rclone/)	 - Show help for rclone commands, flags and backends.
+* [crypt](/crypt/)	 - Info about the crypt overlay
```

Comment:
This is an auto generated file from `rclone cryptdecode --help` so I don't think you can patch it like that alas.

You could put a link in the docs in `cmd/cryptdecode/cryptdecode.go`

Reply:
The source code doesn't appear to contain the `## SEE ALSO` portion of the docs.  How do I add a customized item to that?

Reply:
That is created by cobra I think. I'm not 100% sure how you do that!

I suggest just adding a note at the bottom of the main text.